### PR TITLE
Use cl-lib instead of cl

### PR DESCRIPTION
**Pull request by tarsius**

### CODE REVIEW

File: solarized-theme-utils.el [hunk 1]
```diff
@@ -24,7 +24,7 @@
 ;;
 
 ;;;; Code:
-(require 'cl)
+(require 'cl-lib)
```
Comment (bbatsov) [inline solarized-theme-utils.el]: We also need to add a dep to `cl-lib` from GNU ELPA for older Emacsen (e.g. 24.1).

**Comment by tarsius** [comment]
Ah sorry, did not realize that was distributed with the package. I've added the dependency now.

**Comment by thomasf** [comment]
It's actually not really  meant to be distributed in the package. . 
Thanks!

**Comment by syl20bnr** [comment]
Sorry for hi-jacking this closed PR but I'm curious about this replacement, why `cl-lib` is better than `cl` ? is it just a matter of namespace ?

**Comment by thomasf** [comment]
Yes, cl.el was removed from emacs and replaced by cl-lib a few releases ago.. cl.el is now deprecated and just aliases cl-lib.

**Comment by thomasf** [comment]
To keep compatibility with earlier emcs versions cl-lib is also published to elpa so that emacses which does not have it can download it.. That's the main reason to have it as a package depencency.

**Comment by syl20bnr** [comment]
Thank you!

Le mercredi 16 septembre 2015, Thomas Frössman <notifications@github.com> a
écrit :

> To keep compatibility with earlier emcs versions cl-lib is also published
> to elpa so that emacses which does not have it can download it.. That's the
> main reason to have it as a package depencency.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bbatsov/solarized-emacs/pull/205#issuecomment-140746120>
> .
>


-- 
-syl20bnr-

**Pull request merged**